packages/core/apps/core-apps-server-internal/src/assets/fonts/roboto_mono/LICENSE.txt

```diff
@@ -0,0 +1,202 @@
+
+                                 Apache License
+                           Version 2.0, January 2004
+                        http://www.apache.org/licenses/
+

```

Comment:
(commenting here because GH doesn't allow me to do it on font/binary files): related to my first comment: What's the issue with the assets exactly? Are we forced to duplicate **all** of them (meaning keeping them **all** under `src/core/server/core_app/assets` too), or is the issue just about the scss import of the `src/core/server/core_app/assets/legacy_light_theme.css` file from

https://github.com/elastic/kibana/blob/bebec37f0435f292f19834d49c8db0006f85fcbb/x-pack/plugins/canvas/shareable_runtime/index.ts#L9

?

Because if that's the latest, I'd rather only keep the strict subset of required files in `src/core/server/core_app/assets` instead of duplicating them all.

Reply:
Agreed, the only asset we need to keep for now is `legacy_light_theme.css`. We can follow up with changing the export at a later stage when we're forced to touch other teams' code.